dom0/sd-dom0-qvm-rpc.sls

```diff
@@ -76,7 +76,23 @@ dom0-rpc-qubes.USB:
     - marker_start: "### BEGIN securedrop-workstation ###"
     - marker_end: "### END securedrop-workstation ###"
     - content: |
-        $anyvm $tag:sd-workstation deny
+        sd-export-usb sys-usb allow
+        @anyvm @tag:sd-workstation deny
+dom0-rpc-qubes.ensure.USBAttach:
```

Comment:
I am not sure what this step does, does this ensure the default USBAttach policy is present, and otherwise populates it with the default specified here? If that is the case, we may want to require this step prior to running the `dom0-rpc-qubes.USBAttach` step to prevent that change from being applied *before* this check is performed.

Reply:
Added the `require` to ensure correct ordering.